bin/host/src/interop/cfg.rs

```diff
@@ -228,6 +232,27 @@ impl InteropHost {
         })
     }
 
+    /// Reads the [L1ChainConfig]s from the file system and returns a map of L1 chain ID ->
+    /// [L1ChainConfig]s.
```

Comment:
```suggestion
    /// Reads the [`L1ChainConfig`]s from the file system and returns a map of L1 chain ID ->
    /// [`L1ChainConfig`]s.
```
we really should be code formatting code, like the std library does